src/components/flowchart/styles/_tooltip.scss

```diff
@@ -0,0 +1,63 @@
+@import './variables';
+
+.pipeline-flowchart__tooltip {
+  position: absolute;
+  top: -60px;
+  left: -20px;
+  padding: 12px 20px;
+  color: white;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity ease 0.1s, visibility ease 0.1s;
+  pointer-events: none;
+  z-index: 9;
```

Comment:
Might be worth having z-index in some variables file e.g. $z-lowest, $z-highest - might help z-index issues in the future, what do you think?

Reply:
I don't mind too much either way - it's not something I've done much in the past. I'm torn about whether this is beneficial or detrimental abstraction, or premature/appropriate optimisation?

I'll do some research for appropriate z-index ordering systems. I suspect that a system with names that reference the things themselves, but which can be reordered sensibly (so no numbers or 'lowest'/'highest') might be wise.

Reply:
Definitely should be a consideration if you find yourself using z-indexes elsewhere, the notorious z-index: 9999 is not far behind without putting some structure around it

Reply:
👍 Definitely. That being said, this is the only place in the application where z-index is being used, so I'll leave it for now?